go.mod

```diff
@@ -0,0 +1,163 @@
+module github.com/lndk-org/lndk
+
+go 1.19
+
+// This is the particular commit of Carla's custom feature bits code.
```

Comment:
You can just use [v0.17.0-beta.rc5](https://github.com/lightningnetwork/lnd/releases/tag/v0.17.0-beta.rc5), the last release has this feature in it!

Reply:
Oh awesome, great to see 17 is almost out